cmd/kueuectl/app/list/list_localqueue.go

```diff
@@ -0,0 +1,226 @@
+		},
+	}
+
+	o.PrintFlags.AddFlags(cmd)
+
+	addAllNamespacesFlagVar(cmd, &o.AllNamespaces)
+	addFieldSelectorFlagVar(cmd, &o.FieldSelector)
+	addLabelSelectorFlagVar(cmd, &o.LabelSelector)
+	addClusterQueueFilterFlagVar(cmd, &o.ClusterQueueFilter)
+
+	return cmd
+}
+
+// Complete completes all the required options
+func (o *LocalQueueOptions) Complete(clientGetter genericclioptions.RESTClientGetter, cmd *cobra.Command, args []string) error {
+	var err error
+
+	o.Namespace, o.EnforceNamespace, err = clientGetter.ToRawKubeConfigLoader().Namespace()
+	if err != nil {
+		return err
+	}
+
+	config, err := clientGetter.ToRESTConfig()
+	if err != nil {
+		return err
+	}
+
+	clientset, err := versioned.NewForConfig(config)
+	if err != nil {
+		return err
+	}
+
+	o.Client = clientset.KueueV1beta1()
+
+	defaultPrinter, err := o.PrintFlags.ToPrinter()
+	if err != nil {
+		return err
+	}
+	o.DefaultPrintObj = defaultPrinter.PrintObj
+
+	tablePrinter := printers.NewTablePrinter(printers.PrintOptions{WithNamespace: o.AllNamespaces})
+	o.TablePrintObj = tablePrinter.PrintObj
+
+	return nil
+}
+
+// Validate validates required fields are set to support structured generation
+func (o *LocalQueueOptions) Validate() error {
+	if !o.AllNamespaces && len(o.Namespace) == 0 {
+		return fmt.Errorf("namespace must be specified")
+	}
+	return nil
+}
+
+// Run performs the list operation.
+func (o *LocalQueueOptions) Run(ctx context.Context) error {
+	namespace := o.Namespace
+	if o.AllNamespaces {
+		namespace = ""
+	}
+
+	opts := metav1.ListOptions{LabelSelector: o.LabelSelector, FieldSelector: o.FieldSelector}
+	list, err := o.Client.LocalQueues(namespace).List(ctx, opts)
+	if err != nil {
+		return err
+	}
+
+	o.filterList(list)
+
+	if len(list.Items) > 0 {
+		return o.print(list)
+	}
+
+	if !o.AllNamespaces {
+		fmt.Fprintf(o.ErrOut, "No resources found in %s namespace.\n", o.Namespace)
+	} else {
+		fmt.Fprintln(o.ErrOut, "No resources found")
+	}
+
+	return nil
+}
+
+func (o *LocalQueueOptions) filterList(list *v1beta1.LocalQueueList) {
+	if len(o.ClusterQueueFilter) > 0 {
+		var filteredItems []v1beta1.LocalQueue
+		for _, lq := range list.Items {
+			if lq.Spec.ClusterQueue == v1beta1.ClusterQueueReference(o.ClusterQueueFilter) {
+				filteredItems = append(filteredItems, lq)
+			}
+		}
+		list.Items = filteredItems
+	}
+}
+
+func (o *LocalQueueOptions) print(list *v1beta1.LocalQueueList) error {
+	if len(*o.PrintFlags.OutputFormat) == 0 {
+		return o.printTable(list)
+	} else {
+		return o.DefaultPrintObj(list, o.Out)
+	}
+}
+
+func (o *LocalQueueOptions) printTable(list *v1beta1.LocalQueueList) error {
+	table := &metav1.Table{
+		ColumnDefinitions: localQueueColumnDefinitions(),
+		Rows:              printLocalQueueList(list),
+	}
+	return o.TablePrintObj(table, o.Out)
+}
+
+func localQueueColumnDefinitions() []metav1.TableColumnDefinition {
+	return []metav1.TableColumnDefinition{
+		{Name: "Name", Type: "string", Format: "name"},
```

Comment:
Will it pring namespace if --all-namespacces is given?

Reply:
Yes, Here 9ac6990#diff-f54fd6a8e6601847064d3799bee030e37d44686c04680124f63b9b316d5fce24R126

Reply:
Also we checking it here https://github.com/kubernetes-sigs/kueue/blob/9ac6990cbacc04f639ff8a89de3870f2dde18593/test/integration/kueuectl/list_test.go#L91-L107